pyglmnet/pyglmnet.py

```diff
@@ -14,52 +15,55 @@ def _lmb(distr, beta0, beta, X, eta):
     return _qu(distr, z, eta)
 
 
-def _qu(distr, z, eta):
-    """The non-linearity."""
-    if distr == 'softplus':
+def _qu(distr, z, eta, return_grad=False):
```

Comment:
can we call this function `_mu`? i believe the tutorial started with $q()$ but we started calling it $\mu()$ in the cheatsheet. it would be nice to make the variable names consistent with docs.